.github/workflows/cookiecutter-e2e.yml

```diff
@@ -0,0 +1,67 @@
+name: E2E Cookiecutters
+
+on:
+  push:
+#    branches: [main]
+#    paths: [cookiecutter/**]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+env:
+  FORCE_COLOR: "1"
+
+jobs:
+  lint:
+    name: Lint ${{ matrix.cookiecutter }}, Replay ${{ matrix.replay }} on ${{ matrix.python-version }} ${{ matrix.python-version }} / ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: true
+      matrix:
+        include:
+          - { cookiecutter: "tap-template", replay: "tap-rest-api_key-github.json", python-version: "3.9",  os: "ubuntu-latest" }
+          - { cookiecutter: "target-template", replay: "target-per_record.json", python-version: "3.9",  os: "ubuntu-latest" }
+
+    steps:
+    - name: Check out the repository
+      uses: actions/checkout@v3.3.0
+
+    - name: Setup Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v4.5.0
+      with:
+        python-version: ${{ matrix.python-version }}
+        architecture: x64
+        cache: 'pip'
+        cache-dependency-path: 'poetry.lock'
```

Comment:
Probably doesn't make sense to use the package's `poetry.lock` as cache key for the templates, e.g. the templates cache could need to refreshed even if `poetry.lock` hasn`t changed.
```suggestion
```